config/routes.rb

```diff
@@ -11,8 +11,9 @@
   
   namespace :api do
     namespace :v1 do
-      post :auth, to: "authentication#create"
+      post :auth, to: "authentication#login"
       resources :users do
+        post :create, to: "users#create"
         resources :posts do
           resources :comments
```

Comment:
- Kindly include the following missing routes - /login and /register or /signup

Reply:
Added The routes /login and /signup
`POST   /api/v1/signup`
`POST   /api/v1/login`

![fsdfsdfsdf](https://user-images.githubusercontent.com/83514256/206855526-a248347e-d678-4cba-a43f-56387b2a23d0.PNG)